common/lib/xmodule/xmodule/word_cloud_module.py

```diff
@@ -248,7 +248,8 @@ def get_html(self):
             'submitted': self.submitted,
             'instructions_default': WordCloudFields.instructions.default,
             'instructions': self.instructions,
-            'display_name': self.display_name
+            'display_name': self.display_name,
+            'display_name_default': WordCloudFields.display_name.default
```

Comment:
OK so the only variables available to the underscore template are those passed through context. So, I'd either need to pass the individual defaults (as I was before), or pass in the entire object. I think I'm going to keep it like this for clarity.